recipes/pygame/patches/0006-add-some-more-display-tags-to-tests-modules.patch

```diff
@@ -0,0 +1,759 @@
+From 9f884175a0fda87978a0cd126ae54b272a17ded2 Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Wed, 20 Jul 2022 21:29:46 +0200
+Subject: [PATCH 06/10] add some more 'display' tags to tests & modules
```

Comment:
When I wrote this patch, I wasn't aware of `SDL_VIDEODRIVER` and `SDL_AUDIODRIVER` that @ankith26 mentioned [above](https://github.com/conda-forge/staged-recipes/pull/19665#discussion_r1001587822). It's possible that this patch can be simplified due to that, but I don't think it's necessary for this PR.